src/v3/src/util/theme.ts

```diff
@@ -153,14 +154,26 @@ export const createThemeAndTokens = (
       },
       MuiInputBase: {
         styleOverrides: {
-          root: {
+          root: ({ ownerState }) => ({
             width: '100%',
-          },
+            ...(ownerState.name && isLtrField(ownerState.name) && {
+              direction: 'ltr',
+            }),
+          }),
           input: {
             '::-ms-reveal': {
               display: 'none',
             },
           },
+          adornedStart: ({ ownerState }) => ({
+            // Explicitly switch to physical properties for telephone code
+            ...(ownerState.type === 'tel' && {
+              '& .MuiInputAdornment-root': {
+                marginInlineStart: 0,
+                marginLeft: mergedTokens.Spacing2,
+              },
```

Comment:
Since phone input is always LTR, we need to force the start adornment to always have padding on the left